gazelle/resolve.go

```diff
@@ -90,7 +89,7 @@ func (l *swiftLang) Resolve(
 	if len(unresolved) > 0 {
 		rr.AddUnresolved(unresolved...)
 		log.Printf("Unable to find dependency labels for %v",
-			strings.Join(resResult.Unresolved, ", "))
+			strings.Join(unresolved, ", "))
```

Comment:
This was displaying the wrong list because `resResult.Unresolved` is not modified, only `unresolved` is added to if we cant find in `http_archive` and now in built-ins.